src/core/gosling.schema.ts

```diff
@@ -982,6 +983,7 @@ export interface VectorData {
  */
 export interface BEDDBData {
```

Comment:
Should we change letter cases of interfaces from

`JSONData, CSVData, BIGWIGData, VCFData, BAMData, BEDDBData`

to

`JsonData, CsvData, BigWigData, VcfData, BamData, BeddbData`

consistent with data fetcher class names?

Reply:
I'd be in support of this choice!

Reply:
Okay! I will update this in a follow-up PR.